### DISCUSSION TITLE
Custom Gatsby 404 page with now dev

### DISCUSSION THREAD

**Discussion by jacobgranberry** [vercel/vercel#4267] [General]
Hey there!  Trying to implement a custom 404 page in my gatsby app using now dev.  I've added the example now.json found in the official docs but when I do this, routing breaks and every page displays the vercel 404 page.  Is there something I'm missing here?

`{
  "version": 2,
  "builds": [{ "src": "package.json", "use": "@now/static-build" }],
  "routes": [
    { "src": "/(.*)", "dest": "/$1" },
    { "handle": "filesystem" },
    { "src": "/.*", "status": 404, "dest": "/404" }
  ]
}
`

**Comment by coetry** [comment]
Hi @jacobgranberry can you share a quick repo with us so we can reproduce the error?

**Reply by jacobgranberry** [reply]
Sure thing - latest version of the site is here: https://github.com/WesterosCraft/WesterosCraftSite

**Comment by mrmartineau** [comment]
@jacobgranberry did you solve this issue?

**Comment by styfle** [comment]
We are about to announce this feature very soon.

In the meantime, you won't need any configuration. Instead use the canary channel with `npm install -g vercel@canary` to try it out today.

I tried this with the [gatsby example](https://github.com/zeit/now/tree/master/examples/gatsby) and added a page to `./src/pages/404.js`.

Then I had to click "Preview custom 404 page" after visiting a missing route like the [Gatsby Docs]( https://www.gatsbyjs.org/docs/add-404-page/) mention.
